src/main/java/forestry/Forestry.java

```diff
@@ -56,9 +60,10 @@
 		version = Constants.VERSION,
 		guiFactory = "forestry.core.config.ForestryGuiConfigFactory",
 		acceptedMinecraftVersions = "[1.12]",
-		dependencies = "required-after:forge@[14.21.1.2395,);"
+		dependencies = "required-after:forge@[14.21.1.2424,);"
```

Comment:
I guess that one slipped in there... I need to check when or why I pushed the version. If not required I'll revert that change